extensions/azurePublishNew/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
```

Comment:
Consider making the tsconfig more strict - like the Composer tsconfig.  It will help ensure we write better typescript.